js/coinsbit.js

```diff
@@ -0,0 +1,1114 @@
+            'symbol': symbol,
+            'id': id,
+            'order': orderId,
+            'type': undefined,
+            'takerOrMaker': takerOrMaker,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'cost': cost,
+            'fee': fee,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        if (!price) {
+            throw new InvalidOrder (this.id + ' createOrder needs price field');
+        }
+        request['side'] = side;
+        request['amount'] = this.amountToPrecision (symbol, amount);
+        request['price'] = this.priceToPrecision (symbol, price);
+        const response = await this.privatePostOrderNew (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       orderId: '8629354782',
+        //       market: 'ETH_USDT',
+        //       price: '4000',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640212275.928',
+        //       dealMoney: '7.98856357',
+        //       dealStock: '0.002',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //   }
+        const data = this.safeValue (response, 'result', {});
+        return this.parseOrder (data, market);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        request['orderId'] = id;
+        const response = await this.privatePostOrderCancel (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       orderId: '8631519189',
+        //       market: 'ETH_USDT',
+        //       price: '3100',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640254497.745',
+        //       dealMoney: '0',
+        //       dealStock: '0',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0.002',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //  }
+        const data = this.safeValue (response, 'result');
+        return this.parseOrder (data);
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (since !== undefined) {
```

Comment:
Let's remove this conditional and the exception about `since`. We don't have it anywhere in `fetchOpenOrders`.